lib/blacklistQueries.js

```diff
@@ -0,0 +1,52 @@
+const db = require('./db')
```

Comment:
There's no reason to rename this file. You can revert the name back to blacklist.js